src/tests/utils/render.test-utils.ts

```diff
@@ -2,15 +2,13 @@ import {
   render as svelteRender,
   type RenderResult,
 } from "@testing-library/svelte";
-import { type Component, type ComponentProps } from "svelte";
+import type { Component, ComponentProps } from "svelte";
 
 // prettier-ignore
 // @ts-expect-error Testing-library type not exposed
-import type { MountOptions } from "@testing-library/svelte/types/component-types";
-// prettier-ignore
-// @ts-expect-error Testing-library type not exposed
-import type { ComponentType } from "@testing-library/svelte/types/component-types";
+import type { ComponentType,MountOptions } from "@testing-library/svelte/types/component-types";
```

Comment:
shouldn't this be formatted with a whitespace? I imagine the CI will raise the issue, no?
```suggestion
import type { ComponentType, MountOptions } from "@testing-library/svelte/types/component-types";
```